docs/fundamentals/systems/form/validate.md

```diff
@@ -130,6 +130,32 @@ class EqualsText extends Validator {
 }
 ```
 
+The `execute()` method returns a boolean by default, but it can also be customized to return enums instead. In this example below, we customize the method to return one among some possible enums as strings.
+
+```js
+class EnumOutComeValidator extends Validator {
+  static validatorName = 'EnumOutCome';
+
+  execute(modelValue, maxLength) {
+    if (modelValue.length > maxLength) {
+      return 'Overload';
+    } else if (modelValue.length < 0) {
+      return 'Error';
+    }
+    return 'OK';
+  }
+
+  static async getMessage({ outcome }) {
+    const results = {
+      Overload: 'Too many characters!',
```

Comment:
Looks good! Only thing I would change is lowercase obj keys, that is a good practice. Also we don't have to typecast the outcome